.vib/common/goss/templates/check-added-openssl.yaml

```
@@ -0,0 +1,10 @@
+########################
+# If the OpenSSL non-system package is added
+# it checks it is using the correct version
+# Needed vars:
+#   - .Vars.root_dir
+########################
+command:
+  check-added-openssl:
+    exec: export BITNAMI_ROOT_DIR={{ .Vars.root_dir }} && ./common/goss/scripts/check-added-openssl.sh
```

Just a suggestion. Maybe we can call it `check-bitnami-openssl` instead? WDYT? 
```suggestion
    exec: export BITNAMI_ROOT_DIR={{ .Vars.root_dir }} && ./common/goss/scripts/check-bitnami-openssl.sh
```